docs/orchestration/getting-started/quick-start.md

```diff
@@ -0,0 +1,9 @@
+# Overview
+
+This is a Quick Start guide to help you get a basic Prefect flow registered and running on one of our Prefect Cloud or Prefect Server orchestration layers.  
+
+Because it is a foundational part of running a flow, this guide includes an overview of what is needed to get started with Prefect Core.  If you already have a solid understanding of Core and want to move  onto orchestration, we recommend you go straight to [set up Server or Cloud](/set-up.md).  
```

Comment:
Can the pages for "install" through "basic core flow" live in a single place? It feels messy repeating stuff across Orchestration and Core. And as a user I now need to do work (even with Jenny's very nice guidance in the docs) to figure out where to go after Core

I would lean towards just putting everything needed for the tutorial in Orchestration, which naturally leads into next steps of setting up Cloud/Server, advanced flows, etc

Reply:
Agreed, and then we don't have to worry about catching our own errors twice :)